docs/token/non-standard.md

```diff
@@ -0,0 +1,27 @@
+# Non-standard behaviour and gotchas
+
+Some specific behaviours encoded into ANT may be useful to be aware of, even when not programmatically interacting with the ANT contract.
+
+## Gas
+
+Because ANT includes historical balance records for each address, token transfers are slightly more expensive than some other tokens.
+
+The exact amount per transfer will differ between hardforks and whether the receiver has previously held an ANT balance before, but it is safe to start with a gas limit of 150,000 and optimize lower over time.
+
+## Changing non-zero allowances
+
+The ANT contract reverts when attempting to modify an allowance (calling `ANT.approve()`) that is not zero. [More details of the rationale behind this](https://github.com/ethereum/EIPs/issues/20#issuecomment-263524729).
+
+To successfully change an allowance in this circumstance, you must first ask the user to reset their allowance for that spender to zero (`ANT.approve(0)`), and then to the amount desired.
+
+It useful to note that this behaviour is considered non-standard for ERC20s, and may lead to problems if ANT is used by a contract that is not designed to handle this behaviour.
+
+## Initially vested tokens
+
+Some token holders (founders, advisors, and early contributors) received tokens from the initial token sale that included a vesting period to limit their transferability.
```

Comment:
Advisors and founders got their tokens from the 30% that 'Aragon Dev' got minted when the sale closed. Those allocations were then made from the now AA multisig

Reply:
I've reworded this section in 15fc969.